docs/architecture/grpc-for-wcf-developers/wcf-services-to-grpc-comparison.md

```diff
@@ -69,7 +69,7 @@ namespace HelloGrpc
 }
 ```
 
-This chapter will refer to this example code when explaining various concepts and features of gRPC.
+This chapter will refer to this example code when explaining concepts and features of gRPC.
```

Comment:
Perhaps we can say different?

```suggestion
This chapter will refer to this example code when explaining different concepts and features of gRPC.
```

Reply:
Using "different" is against the Microsoft Writing Style Guide (https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33638), though Acrolinx prefers it over "various." I tend to remove either word when it doesn't seem to add value. Your suggestion is a small change, and I don't mind making it.